### DISCUSSION TITLE
TFLint annotation issue

### DISCUSSION THREAD

**Discussion by engg-abhishekshinde** [terraform-linters/tflint#1955] [Q&A]
I have followed TFLints document to disable TFLints error on specific line. But it's not working.

Document link which I have followed - https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/annotations.md

Source Code -

```
module "ecr" {
  source = "../../../ecr"

  name = format("%s-%s", var.name, var.env)
  // tflint-ignore: terraform_unused_declarations, all, other_rule
  tags = var.tags
}
```

Error log -

```
Run tflint --color --recursive --minimum-failure-severity=error
Failed to prepare rule checking; aws/applications/aberration-detection/ecr/ecr.tf:6,10-18: Reference to undeclared input variable; An input variable with the name "tags" has not been declared. This variable can be declared with a variable "tags" {} block. working_dir=aws/applications/aberration-detection/ecr:

Error: Reference to undeclared input variable

  on aws/applications/aberration-detection/ecr/ecr.tf line 6, in module "ecr":
   6:   tags = var.tags

An input variable with the name "tags" has not been declared. This variable can be declared with a variable "tags" {} block.
```

**Comment by wata727** [comment]
Annotations only work on issues. Runtime errors cannot be ignored.

**Reply by engg-abhishekshinde** [reply]
I'm running following command - 

`tflint --color --recursive --minimum-failure-severity=error.`

What is meaning of runtimes error?. How I can ignore following error?.

`Reference to undeclared input variable; An input variable with the name "tags" has not been declared. This variable can be declared with a variable "tags" {} block`

**Reply by wata727** [reply]
TFLint will raise an error if it cannot be interpreted as a Terraform configuration.

To fix this error, you need to modify the file as instructed in the error message and ensure that `terraform validate` and `terraform plan` passes successfully.

**Reply by engg-abhishekshinde** [reply]
Thanks. Got it.. :)
